news/1562.bug

```diff
@@ -0,0 +1 @@
+Context navigation @contextnavigation endpoint is not going children in depth 
```

Comment:
@andreiggr could you try to formulate the changelog entry in a way that it is obvious to the reader what we fixed here?